final_task/pycalc/addmultsigns.py

```diff
@@ -0,0 +1,77 @@
+"""This module contains a class that allows to take into account implicit multiplication signs"""
+
+# import from pycalc self library
+from .pycalclib import constants, functions, negative_functions
+
+
+class Multsignsadder():
+    """A model of mult_signs_adder capable of adding implicit multiplications signs in list of tokens"""
+
+    def __init__(self, tokens):
+        """Initialize mult_signs_adder"""
+        self.tokens = tokens
+        self.extended_tokens = []
+        self.constants = constants
+        self.functions = functions
+        self.negative_functions = negative_functions
+
+    def is_number(self, token):
```

Comment:
This method can be `staticmethod` or just a util function. `self` attribute is not needed inside the function.

Reply:
moved to utils.py as a function